app/views/render_async/_request_jquery.js.erb

```diff
@@ -18,7 +17,9 @@ if (window.jQuery) {
         data: "<%= escape_javascript(data.to_s.html_safe) %>",
         headers: headers
       }).done(function(response) {
-        <% if interval %>
+          lazyLoaded(document.getElementById("<%= container_id %>"));
```

Comment:
We could check whether `lazy_load` is turned on before marking an element as lazy loaded?

This will mark each element as lazy loaded even if the user is not using lazy load feature, right?